crates/core/tedge/src/cli/certificate/create.rs

```diff
@@ -29,6 +29,7 @@ impl Command for CreateCertCmd {
     fn execute(&self) -> anyhow::Result<()> {
         let config = NewCertificateConfig::default();
         self.create_test_certificate(&config)?;
+        println!("Certificate was successfully created.");
```

Comment:
The punctation does not provide much value, so I would recommend to remove it.
```suggestion
        println!("Certificate was successfully created");
```